apps/openmw/mwrender/npcanimation.cpp

```diff
@@ -264,6 +264,15 @@ static NpcAnimation::PartBoneMap createPartListMap()
     result.insert(std::make_pair(ESM::PRT_LPauldron, "Left Clavicle"));
     result.insert(std::make_pair(ESM::PRT_Weapon, "Weapon Bone"));
     result.insert(std::make_pair(ESM::PRT_Tail, "Tail"));
+
+    // These ones supposed to store the holstered weapons, as in later TES games, but without scabbard
```

Comment:
"These are for storing sheathed weapons (no scabbard).

I'd avoid copyrighted IP references, considering OpenMW isn't supposed to be Morrowind-only. :P

Side note: scabbard support in later TES games is pluginless: the engine searches for a shape named scb (I assume "scabbard" alias would also work if it could be implemented in openmw) in the mesh and if it finds one, it assumes it's the scabbard and leaves it on its place when the player unsheathes the weapon. Could be a cool feature to replicate, while you are still working on this.

Reply:
So in this case I need a way to attach just a node instead of whole model. Interesting.